GitHubLabels.ps1

```diff
@@ -565,6 +628,121 @@ function Set-GitHubLabel
     }
 }
 
+function Add-GitHubLabel
+{
+<#
+    .DESCRIPTION
+        Adds a label to an issue in the given GitHub repository.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Issue
+        Issue number to add the label to.
+
+    .PARAMETER LabelName
```

Comment:
We use `Name` as the parameter name for all of the other Label functions, with it _aliased_ to be `LabelName`.